rednotebook/gui/editor.py

```diff
@@ -131,8 +131,8 @@ def highlight(self, text):
         buf.remove_tag_by_name("highlighter", start, end)
 
         # Highlight matches
-        if text:
-            for match_start, match_end in self.iter_search_matches(text):
+        for query in enumerate(queries):
+            for match_start, match_end in self.iter_search_matches(query):
```

Comment:
`query` is now a tuple, e.g., `(0, "cat")`. Doesn't that break the call to iter_search_matches? To me it seems the `enumerate` should be removed.

Reply:
Oops, you're right. I added an `enumerate` when I was testing something and forgot to remove this. On it now.